src/components/header/header.actions.js

```diff
@@ -0,0 +1,2 @@
+export const addTodo = todo => ({ type: 'ADD_NEW_TODO', payload: todo })
+export const setNewTodo = todo => ({ type: 'SET_NEW_TODO', payload: todo })
```

Comment:
`CHANGE_NEW_TODO`

Reply:
What about using redux-form ?

Reply:
- CHANGE_TODO its for edit an existing TODO, and NEW_TODO its the name of input.
- I find that the goal is to show our libraries and redux-form adds yet a level of abstraction and makes it more difficult to understand